### PR TITLE
240: Add section in Common Issues for manually deleting Layer0 instances

### LINKED ISSUE #240: Manual Resource Deletion How-To
We should probably provide our users with an easy list of the resources that Layer0 creates in AWS and how they can go about removing them manually if shit goes too far south for Layer0 or Terraform to save them.

Here's a list that was well-received when I posted it in the public xfra channel - it needs some updating (like RDS -> DynamoDB):

```
MANUAL AWS DELETION

EC2 CONTAINER SERVICE
- delete the cluster

EC2
- Auto Scaling Groups
- Launch Configurations
- Instances
- Loadbalancers

RDS
- DB Instances
    - don’t make a final snapshot
- Subnet Groups

IAM
- Groups
- Users
- Roles

S3
- search for prefix

VPC
- VPCs
    - don’t worry about Connection box
- Security Groups

CLI
- aws iam delete-instance-profile --instance-profile-name [name]
- aws iam delete-server-certificate --server-certificate-name [name]
- find these names by running `aws iam list-instance-profiles` and `aws iam list-server-certificates`
```

### PR DESCRIPTION
**Pull request by jparsons04**
**What does this pull request do?**
Adds a section to the Troubleshooting>Common Issues section of the documentation that describes what resources are created when a Layer0 instance is created and how they can be manually removed.

**How should this be tested?**
`make build && make serve` from the `docs-src/` directory

closes #240

### CODE REVIEW

**Comment by sesh-kebab** [comment]
This is really good - very concise.

File: docs-src/docs/troubleshooting/commonissues.md [hunk 1]
```diff
@@ -1,5 +1,73 @@
 #Common issues and their solutions
 
+##Manually deleting a Layer0 instance from AWS
+
+Sometimes, your Layer0 instance might get into an unresponsive and unrecoverable state. This section describes the AWS resources that are created when you create a Layer0 instance and what resources needs to be removed by hand in such an event.
```
Comment (diemonster) [inline docs-src/docs/troubleshooting/commonissues.md]: At the risk of impact the conciseness of this, having a better definition around 'unresponsive' and 'unrecoverable' would be wise. For example, there are some cases where simply re-creating the API is enough to get things working again, in which case these manual steps wouldn't be needed.

Comment (jparsons04) [reply docs-src/docs/troubleshooting/commonissues.md]: I could add some tips on commands to run to refresh the instance, like running `l0-setup plan`/`l0-setup apply` to rebuild the instance.

Comment (jparsons04) [reply docs-src/docs/troubleshooting/commonissues.md]: :white_check_mark:

File: docs-src/docs/troubleshooting/commonissues.md [hunk 2]
```diff
@@ -1,5 +1,86 @@
...
+    * Subnets
+        * 3 Public subnets: `l0-<prefix>-subnet-public-<region & availability zone>`
+        * 3 Private subnets: `l0-<prefix>-subnet-private-<region & availability zone>`
+    * Route Tables
+        * A blank default
+        * `l0-<prefix>-rt-private`
+        * `l0-<prefix>-rt-public`
+    * Internet Gateway: `l0-<prefix>-igw`
+    * NAT Gateway: nameless but associated with the VPC
+    * Network ACL: nameless but assocaited with the VPC
```
Comment (tlake) [inline docs-src/docs/troubleshooting/commonissues.md]: [typo] associated

File: docs-src/docs/troubleshooting/commonissues.md [hunk 3]
```diff
@@ -1,5 +1,86 @@
...
+* CloudWatch
+    * Log Group: `l0-<prefix>`
+
+* DynamoDB
+    * Tables
+        * `l0-<prefix>-lock`
+        * `l0-<prefix>-tags`
+
+
+Most resources can be removed through the AWS Console, but some need to be removed from the AWS CLI
```
Comment (diemonster) [inline docs-src/docs/troubleshooting/commonissues.md]: [nit] missing a period here. We might want to reconsider this statement as well, it's pretty ambiguous as-is. Perhaps just using the CLI is the best way to go as it provides the most uniform experience.

**Comment by jparsons04** [comment]
I fixed the BOM typesetting inconsistency. I'm leaving the commented out stuff at the bottom because I'm not currently sure about its current impact.

**Pull request merged**